<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterphoneChatWidget</class>
 <widget class="QWidget" name="InterphoneChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>65</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InterphoneChatWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0">
   <property name="spacing">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelIcon">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>icon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTip">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="StylePushButton" name="pushButtonAdd">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="StylePushButton" name="pushButtonOpen">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="StylePushButton" name="pushButtonQuit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StylePushButton</class>
   <extends>QPushButton</extends>
   <header location="global">stylepushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
